package.json

```diff
@@ -11,9 +11,6 @@
     "url": "https://github.com/hoodiehq/hoodie/issues"
   },
   "dependencies": {
-    "@hoodie/admin": "^1.0.1",
-    "@hoodie/client": "^10.0.0",
-    "@hoodie/server": "^23.0.0",
```

Comment:
you need to put these back in, that’s why the tests are currently failing, see https://travis-ci.org/hoodiehq/hoodie/builds/249421274#L564